src/libraries/Microsoft.VisualBasic.Core/tests/FileSystemTests.cs

```diff
@@ -23,12 +23,11 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
-        // On OSX, the temp directory /tmp/ is a symlink to /private/tmp, so setting the current
+        // On OSX/MacCatalyst, the temp directory /tmp/ is a symlink to /private/tmp, so setting the current
```

Comment:
Can `/tmp` be a symlink on other OSes too? Would it be better for the test to check whether `/tmp` is a symlink and skip the validation as appropriate? It would save us from disabling the test one OS at a time.

Reply:
That sounds better approach but I couldn't find how to check whether /tmp is a symlink.

Reply:
> That sounds better approach but I couldn't find how to check whether /tmp is a symlink.

Console.WriteLine(((File.GetAttributes("/tmp") & FileAttributes.ReparsePoint) != 0) ? "Symlink" : "Not a symlink");